scylla-cql/src/_macro_internal.rs

```diff
@@ -148,5 +156,73 @@ pub mod ser {
                 partial.check_missing()
             }
         }
+
+        pub struct ByColumn<'i, 'c> {
+            columns: std::slice::Iter<'i, ColumnSpec<'c>>,
+        }
+
+        impl ByColumn<'_, '_> {
+            #[inline]
+            pub fn next<'b, T, const ENFORCE_NAME: bool>(
+                &mut self,
+                expected: &str,
+                value: &impl SerializeValue,
+                writer: &'b mut RowWriter<'_>,
+            ) -> Result<WrittenCellProof<'b>, SerializationError> {
```

Comment:
:wrench: `next` is commonly associated with `Iterator`, as a method that simply returns consecutive items.

Reply:
```suggestion
            pub fn serialize_next_column<'b, T, const ENFORCE_NAME: bool>(
                &mut self,
                expected: &str,
                value: &impl SerializeValue,
                writer: &'b mut RowWriter<'_>,
            ) -> Result<WrittenCellProof<'b>, SerializationError> {
```

Reply:
I renamed the struct to be `NextColumnSerializer` and this function to `serialize` which I think already implies "next" given the name of the struct. Let me know if you think that is enough.

Reply:
It's OK for me.